app/plugins/Files/js/loaders.js

```diff
@@ -76,7 +89,9 @@ function uploadDir(dirPath, nickname) {
 					notify('Cannot read ' + file, 'error');
 					return;
 				}
-				if (!isUnixHiddenPath(filePath) && stats.isFile()) {
+				if (file.slice(-4) === '.sia') {
+					loadDotSia(filePath);
```

Comment:
when in doubt, go with the least surprising behavior. Doesn't mean we can't have this functionality, just that it needs to be clear to the user.